Billing & Subscription documentation (#431)
bufbuild/docs.buf.build: docs/subscription/faq.md

---
id: faq
description: Subscription & Billing FAQs
keywords: [Buf, Buf Schema Registry, billing FAQs, pull rate limiting FAQs, subscription, platform]
title: FAQs
hide_table_of_contents: true
---

---

## Subscription tiers

### What subscriptions levels are available for Buf?

> [Buf Subscriptions](#) come in three levels:

Reviewer: ```suggestion
[Buf Subscriptions](#) come in three tiers:
```

using tiers to be consistent with industry language

* **Free plans** for both individuals and development teams that include unlimited public repos.
* **Buf Team subscriptions**
* **Buf Professional subscriptions**
* **Buf Enterprise**

### What are the differences between Free, Professional and Team Subscriptions?

> Professional and Team subscriptions feature additional features, support, consumption levels and more. You can compare

Reviewer: ```suggestion
Professional and Team subscriptions feature additional features, support, consumption levels, and more. You can compare
```

programs at the [Buf Pricing](https://www.buf.build/pricing) page.

### What are the rate limits for free subscriptions?

Reasonable rate limits apply to all Buf Schema Registry traffic regardless of pricing tier.

### Is it possible to enable access for our organization's IP addresses instead of tracking and managing individual accounts for our numerous agents, users, and seats?

> Our Enterprise package includes an Isolated Single-Tenant instance, which enables large organizations to use Buf Schema

Reviewer: ```suggestion
Our Enterprise package includes an isolated single-tenant instance, which enables large organizations to use Buf Schema
```

Buf is a proud open source software company, and provides qualifying OSS software projects with access to Buf Schema
Registry with unlimited public repositories, unlimited consumption generated client and server libraries, and unlimited
collaborators at no cost. For questions about open source projects and
pricing, [Contact Us](https://www.docs.buf.build/contact-us).

---

## Renewals, expansion, and account changes

### Do I need to do anything at the end of my subscription term?

No. All subscriptions are automatically renewed at the end of the term using the original form of payment.

### What happens if I pay for a monthly subscription and add more types?

When you add types to your organization in the middle of your billing cycle, you will be charged for the greater number
of types in the following month, providing you with the current months grace.

### How do I upgrade to a Professional plan?

Upgrading your plan to a Professional plan offers you unlimited public and private repos. For information on how to
upgrade to a Professional plan from a legacy plan, see the [How To Upgrade](manage.md#upgrade) page.

### How do I upgrade to a Team plan?

> Upgrading your legacy plan to a Team plan offers you unlimited private repos.For information on how to upgrade to a Team

Reviewer: ```suggestion
Upgrading your legacy plan to a Team plan offers you unlimited private repos. For information on how to upgrade to a Team plan,
```

see the [How To Upgrade](manage.md#upgrade) page.

### How do downgrades from a Professional or Team plan work?

When you downgrade your Professional or Team plan, changes are applied at the end of your billing cycle. For example, if
you are currently on a Team plan which is billed on the 8th of every month, and you choose to downgrade to a Free Team
plan on the 15th, your Team plan will be active until 7th of the following month. You will be transferred to a Free Team
plan on the 8th of the following month. Please refer to the next two questions below for instructions on how to
downgrade your account.

### How do I downgrade from a Team plan to a Free Team plan?

Before you downgrade to a Free plan, ensure that your organization details are updated to reflect features available in
the Free plan. For example, you may need convert any private repositories to public repositories. For information on
what’s included in the Free plan, see the [billing](index.md) page.

### How do I downgrade from Professional to a Free plan?

Before you downgrade to a Free plan, ensure that your account organization details are updated to reflect features

> available in the Free plan. For example, you may need to convert any private repositories to public repositories. For

Reviewer: ```suggestion
available in the Free plan. For example, you may need to convert existing private repositories to public repositories. For
```

information on what’s included in the Free plan, see the [billing](manage.md#downgrade) page.

### How do I add a member to a team in my organization?

For information on how to add a member to a team, see [Add a member to a team](#).

---

## Billing questions

### How can I get an invoice to purchase a Buf Team subscription?

> Enterprise accounts can contact the Buf team for a proposal and invoice. For Team & Professional accounts, purchasing

Reviewer: Would we want to add a contact link here?

via credit card is the only option.

### How do I get a receipt for my subscription?

> Customers will receive an email invoice each monthly billing period. A subscription confirmation with a link to

Reviewer: ```suggestion
Customers will receive an email invoice each monthly billing period and a subscription confirmation with a link to
```

their [billing plan](#) page. From here you can navigate to the [billing history](#) page to download invoices.

### Can I subscribe with a purchase order?

Enterprise accounts can contact the Buf team for a proposal and invoice. For Team & Professional accounts, purchasing
via credit card is the only option.

### I have more questions regarding the new pricing, who can I reach out to?

If you have any questions that are not covered in the FAQ, please reach out and a member of the Buf team will get back
to you.

### How does Buf Professional Subscription pricing work?

> Team starts at $100 USD per month for the first 100 types and each bucket of 100 types thereafter

Reviewer: ```suggestion
Professional starts at $100 USD per month for the first 100 types and each increment of 100 types thereafter
```

### How does Buf Team Subscription pricing work?

> Team starts at $50 USD per month for the first 100 types and each bucket of 100 types thereafter.

Reviewer: ```suggestion
Team starts at $50 USD per month for the first 100 types and each increment of 100 types thereafter.
```

### I have more questions regarding the new pricing, who can I reach out to?

If you have any questions that are not covered in the FAQ, [please reach out](https://www.buf.build/company/contact) and
a member of the Buf team will get back to you.

### Where can I view my billing date?

Navigate to the **billing** tab in your billing settings....

### What payment methods are accepted for Buf Schema Registry?

- American Express
- China UnionPay (CUP)
- Discover & Diners
- Japan Credit Bureau (JCB)
- Mastercard
- Visa

### What happens if my payment fails?

If your payment fails, there is a grace period of x days, including the due date. Buf retries to
collect the payment x times using the following schedule:

- ...

> Buf also sends an email notification `Action Required - Credit Card Payment Failed` with an attached unpaid invoice

Reviewer: ```suggestion
Buf also sends an email notification with the subject `Action Required - Credit Card Payment Failed`, with an attached unpaid invoice
```

after each failed payment attempt.

Once the grace period is over and the invoice is still not paid, the plan is downgraded to a free plan and all
paid features are disabled.

### What billing-related emails will I receive from Buf?

Buf sends the following billing-related emails:

> - Confirmation of paid invoices.

Reviewer: ```suggestion
- Confirmation of paid invoices.
- Notification of payment failures
- Usage overage alerts
```

### How will I be billed for Buf Schema Registry?

Buf Schema Registry bills on a monthly basis, based on the pricing tier you have selected.

### Will I receive an invoice for my Buf Schema Registry subscription?

Yes, you will receive a monthly invoice for your subscription, which will be emailed to the billing email address
associated with your account.

### Can I change my pricing tier at any time?

Yes, you can change your pricing tier at any time by logging into your account and selecting the new pricing tier.

### Will I receive a prorated refund if I downgrade my pricing tier?

No, Buf Schema Registry does not offer prorated refunds for downgrading your pricing tier. Your new pricing tier will
take effect at the beginning of your next billing cycle.

### What happens if I exceed the number of types allowed in my pricing tier?

Types are billed in increments of 100, and the cost per Type depends on the chosen plan. Usage is calculated as the
highest count of Types that existed for an organization over the billing cycle. Buf will send an email alert to

> organization administrators when usage has progressed beyond the previously billed 100-Type bucket. Additionally, you

Reviewer: ```suggestion
organization administrators when usage has progressed beyond the previously billed 100-Type increment, Additionally, you
```
